### ISSUE TITLE
Issue about cross entropy loss in tensorflow

### ISSUE THREAD

**Issue by deepmo24** [WeitaoVan/L-GM-loss#12]
Hi, 
The cross entropy loss you used in Tensforflow is ['sparse_softmax_cross_entropy_with_logits()'.](https://github.com/WeitaoVan/L-GM-loss/blob/598582f0631bac876b3eeb8d6c4cd1d780269e03/tensorflow/cifar100_train.py#L180)  however, this loss function will perform a 'softmax' on 'logits'. Since your lgm loss doesn't contain a 'softmax' operation,  I wonder whether the loss function is correctly used here?

I am looking forward to hearing from you.
